### ISSUE TITLE
Getting promise errors in log when using "disable" feature to disable a specific camera

### ISSUE THREAD

**Issue by matellis** [hjdhjd/homebridge-unifi-protect2#34]
When attempting to use the `disable` feature, I start to see `UnhandledPromiseRejectionWarning` in the log.

Configuration reads (params obfuscated):

```
        {
            "platform": "UniFi Protect",
            "options": [
                    "Disable.FCECDA123456"
            ],
            "controllers": [
                {
                    "address": "172.16.99.5",
                    "name": "UniNVR",
                    "password": "somelongpassphraseabout30charslong",
                    "username": "homebridge"
                }
            ]
        }
```

When I add the options section I get the errors below, when I remove them they go away. I've tried lower case `disable` as well, same effect.

```
(node:4353) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'accessory' of undefined
    at WebSocket.<anonymous> (/usr/local/lib/node_modules/homebridge-unifi-protect2/src/protect-nvr.ts:264:62)
    at WebSocket.emit (events.js:311:20)
    at Receiver.receiverOnMessage (/usr/local/lib/node_modules/homebridge-unifi-protect2/node_modules/ws/lib/websocket.js:797:20)
    at Receiver.emit (events.js:311:20)
    at Receiver.dataMessage (/usr/local/lib/node_modules/homebridge-unifi-protect2/node_modules/ws/lib/receiver.js:437:14)
    at Receiver.getData (/usr/local/lib/node_modules/homebridge-unifi-protect2/node_modules/ws/lib/receiver.js:367:17)
    at Receiver.startLoop (/usr/local/lib/node_modules/homebridge-unifi-protect2/node_modules/ws/lib/receiver.js:143:22)
    at Receiver._write (/usr/local/lib/node_modules/homebridge-unifi-protect2/node_modules/ws/lib/receiver.js:78:10)
    at doWrite (_stream_writable.js:441:12)
    at writeOrBuffer (_stream_writable.js:425:5)
    at Receiver.Writable.write (_stream_writable.js:316:11)
    at TLSSocket.socketOnData (/usr/local/lib/node_modules/homebridge-unifi-protect2/node_modules/ws/lib/websocket.js:872:35)
    at TLSSocket.emit (events.js:311:20)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at TLSSocket.Readable.push (_stream_readable.js:209:10)
    at TLSWrap.onStreamRead (internal/stream_base_commons.js:186:23)
(node:4353) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:4353) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:4353) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'accessory' of undefined
    at WebSocket.<anonymous> (/usr/local/lib/node_modules/homebridge-unifi-protect2/src/protect-nvr.ts:264:62)
    at WebSocket.emit (events.js:311:20)
    at Receiver.receiverOnMessage (/usr/local/lib/node_modules/homebridge-unifi-protect2/node_modules/ws/lib/websocket.js:797:20)
    at Receiver.emit (events.js:311:20)
    at Receiver.dataMessage (/usr/local/lib/node_modules/homebridge-unifi-protect2/node_modules/ws/lib/receiver.js:437:14)
    at Receiver.getData (/usr/local/lib/node_modules/homebridge-unifi-protect2/node_modules/ws/lib/receiver.js:367:17)
    at Receiver.startLoop (/usr/local/lib/node_modules/homebridge-unifi-protect2/node_modules/ws/lib/receiver.js:143:22)
    at Receiver._write (/usr/local/lib/node_modules/homebridge-unifi-protect2/node_modules/ws/lib/receiver.js:78:10)
    at doWrite (_stream_writable.js:441:12)
    at writeOrBuffer (_stream_writable.js:425:5)
    at Receiver.Writable.write (_stream_writable.js:316:11)
    at TLSSocket.socketOnData (/usr/local/lib/node_modules/homebridge-unifi-protect2/node_modules/ws/lib/websocket.js:872:35)
    at TLSSocket.emit (events.js:311:20)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at TLSSocket.Readable.push (_stream_readable.js:209:10)
    at TLSWrap.onStreamRead (internal/stream_base_commons.js:186:23)
(node:4353) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
```

**Comment by hjdhjd** [comment]
Thanks, I'll look into this shortly.

**Comment by hjdhjd** [comment]
This is fixed in v3.1.2, just released.

I debated changing the way this feature works actually. On the one hand, it makes no sense to selectively disable controllers in the options string. To disable a controller, just remove it from the configuration. On the other, I can see a scenario where people want to maintain similar configurations easily and alter their visibility only by using feature options. For now, I've decided to keep the former approach, though I reserve the right to change this in the future. :smile:

I don't want to add unnecessary complexity to configuring this plugin and turn it into a maintenance nightmare. I was clearly trying to be too smart for my own good when I designed this part. :smile:
